.changeset/unlucky-files-smoke.md

```diff
@@ -0,0 +1,5 @@
+---
+"@firebase/firestore": patch
+---
+
+Fixed an issue that may have caused the SDK to raise incomplete snapshots when backgrounded during query execution.
```

Comment:
"may have caused" sounds to me like you are unsure that this is the root cause; however, it seems that the root cause is definitively known. Would it make sense to re-word this to imply that the buggy behavior is intermittent?

e.g. Fixed an issue that causes the SDK to sometimes raise incomplete snapshots...

(this comment was copied from the corresponding change to the Android SDK: https://github.com/firebase/firebase-android-sdk/pull/3267#discussion_r778347911)

Reply:
Done